package.json

```diff
@@ -52,8 +52,7 @@
   "pre-commit": [
     "format_check",
     "lint",
-    "validate",
-    "test"
```

Comment:
please don't forget to revert this change before merging.

a trick I've found is to `git add` all the files you'd like to commit, then make this change, then `git commit`.

that's an effective way to bypass the check which ensures the tests pass without actually committing the change to `package.json`

Reply:
you can also run `git commit --no-verify]` to [skip the pre-commit hooks](https://stackoverflow.com/a/7230886). Not recommended, but useful in a pinch :)

Reply:
Thnx for your suggestions! Hadn't come across this coding pattern before, so I brute-forced my code through... 😳 Will try to be more gentle next time